### DISCUSSION TITLE
自定义主页预设投稿

### DISCUSSION THREAD

**Discussion by cola0908** [Meloong-Git/PCL#4605] [自定义主页]
服务器的自定义主页 有一些快捷功能
![222](https://github.com/user-attachments/assets/2d24950b-37d9-4e3e-a01b-38ea06c9b9f2)
求求了

**Comment by cola0908** [comment]
就叫生电小镇服务器快捷更新面板

**Reply by wuliaodexiaoluo** [reply]
>就叫生电小镇服务器快捷更新面板

这感觉和简单主页比没啥可比性，按龙猫的说法就是质量太低
而且页面链接了第三方广告性质的链接，甚至还有无法保证的 QQ 群，这种建议作为联网更新主页而非预设使用
单独的服务器公告用的自定义主页是几乎不可能被加入预设的

**Comment by WForst-Breeze** [comment]
PCL 预设不会收录有服务器或疑似第三方广告性质的预设主页

> 不包含服务器、第三方网站的广告性质链接，间接广告也不行

**Comment by Deep-Dark-Forest** [comment]
虽然无法被添加至 PCL 预设，不过可以给您服务器的用户分发这个主页（

**Comment by CreeperIsASpy** [comment]
先不说宣传第三方内容的问题
您这个主页实用性真的不高
